RPIPs/RPIP-23.md

```diff
@@ -13,23 +13,23 @@ vote-result: Passed
 ---
 
 ## Abstract
-It would be useful to more explicitly define the pDAO in terms of who they are, what they are
+It would be useful to more explicitly define the Protocol DAO (pDAO) in terms of who they are, what they are
 expected to do, and how they are expected to do it. This document is intended to be a foundational
 definition document that will serve (a) the pDAO itself in answering "what's the right thing to do"
 in a given situation, and (b) the greater community in understanding "what should we expect" and
 "what shouldn't we expect" from the pDAO.
 
 ## Specification
-- Superseding this RPIP requires a 2/3 or greater supermajority pDAO vote
+Superseding this RPIP requires a 2/3 or greater supermajority pDAO vote
 
 ### pDAO Values
-1. The pDAO SHALL serve the RP community at large; this comprises rETH holders, NOs, and RPL holders
+1. The pDAO SHALL serve the Rocket Pool community at large; this comprises rETH holders, Node Operators, and RPL holders
 1. The pDAO SHALL be defined as Node Operators, with power based on effectively staked RPL
-1. The pDAO SHALL actively govern the protocol in the interest of the RP community at large
+1. The pDAO SHALL actively govern the protocol in the interest of the Rocket Pool community at large
+1. The pDAO SHALL prioritize protocol safety
```

Comment:
Please revert the re-ordering - it was quite intentional. Voting is extremely important (since it's how the pDAO acts). I understand the desire to group all the SHALLs, but here the voting one is a SHOULD only due to enforceability not lack of importance.